lib.js

```diff
@@ -232,6 +245,7 @@ module.exports = class ContribCat {
 
 	runPlugins(users) {
 		var result = {
+			startDate: moment().endOf("day").subtract(this.config.days, "days"),
```

Comment:
this is now reportDays, obviously wouldn't have been picked up in the merge of conflicts. I'll update this.